src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 if values < 15`, () => {
+    expect(getHumanAge(13, 5))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if values >= 15 and < 24`, () => {
```

Comment:
The test description does not match the functionality. It says that function should return 1 if values are equal or greater than 15 and less than 24, but the function should return an array with cat's and dog's ages in human years.

Reply:
Done